src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -804,8 +804,43 @@ protected bool TakeStartLine(SocketInput input)
                 QueryString = queryString;
                 HttpVersion = httpVersion;
 
-                bool caseMatches;
+                if (requestUrlPath.Length > 0 && requestUrlPath[0] != '/')
+                {
+                    int hostIndex;
+                    if (requestUrlPath.StartsWith("http://"))
```

Comment:
I'm guessing the normal path is a relative URI this path might not even showup. Is that correct?

---
In reply to: [57374946](https://github.com/aspnet/KestrelHttpServer/pull/711#discussion_r57374946) [](ancestors = 57374946)